src/games/gameitem.py

```diff
@@ -161,38 +157,34 @@ def announceHosting(self):
             self.client.forwardLocalBroadcast(self.host, 'is hosting ' + self.mod + ' <a style="color:' + self.client.getColor("url") + '" href="' + url.toString() + '">' + self.title + '</a> (on "' + self.mapdisplayname + '")')
 
     def update(self, message, client):
-        '''
-        Updates this item from the message dictionary supplied
-        '''
-        self.client  = client
 
-        self.title = message['title']
-        self.host = message['host']
+        # Updates this item from the message dictionary supplied #
 
-        if 'host_id' in message:
-            self.hostid = message['host_id']
-        else:
-            self.hostid = self.client.players.getID(self.host)
+        self.client = client
+
+        if self.title is None:  # you can't change this...
```

Comment:
Yet, there's nothing stopping us from adding that functionality. So why arbitrarily limit it here?